android/src/main/java/com/apparence/camerawesome/CameraPicture.java

```diff
@@ -223,6 +288,26 @@ private void writeToFile(ByteBuffer buffer, File file) throws IOException {
         }
     }
 
+    private File createDummyFile(String extension) {
+        return new File(context.getFilesDir(), "dummy." + extension);
+    }
+
+    private MediaRecorder createRecorder(Surface surface, String filePath) {
+        MediaRecorder mediaRecorder = new MediaRecorder();
+        mediaRecorder.setAudioSource(MediaRecorder.AudioSource.MIC);
+        mediaRecorder.setVideoSource(MediaRecorder.VideoSource.SURFACE);
+        mediaRecorder.setOutputFormat(MediaRecorder.OutputFormat.MPEG_4);
+        mediaRecorder.setOutputFile(filePath);
+        mediaRecorder.setVideoEncodingBitRate(RECORDER_VIDEO_BITRATE);
+        mediaRecorder.setVideoFrameRate(30);
```

Comment:
Can we make a constant out of 30?

```suggestion
        mediaRecorder.setVideoFrameRate(RECORDER_VIDEO_FRAME_RATE);
```